tests/test-cases/Group24-Multi-VCH/24-02-Multi-VCH-Delete.robot

```diff
@@ -0,0 +1,61 @@
+# Copyright 2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 24-02 - Verify vic-machine delete only removes own cVMs
+Resource  ../../resources/Util.robot
+Suite Teardown  Cleanup VCHs
+
+*** Keywords ***
+Cleanup VCHs
+    Run Keyword And Ignore Error  Cleanup VIC Appliance On Test Server
+    Set Environment Variable  VCH-NAME  ${old-vm}
+    Set Environment Variable  BRIDGE_NETWORK  ${old-vch-bridge}
+    Set Environment Variable  VCH-PARAMS  ${old-vch-params}
+    Cleanup VIC Appliance On Test Server
```

Comment:
Might be useful to add a `Run Keyword And Continue On Error` here, as this step should definitely happen and if someone comes along to add more steps they will not need to know to add it. It should not affect the way that line works currently either way :)

Reply:
Sure, that makes sense. It's a no-op for now.